sdks/go/test/integration/io/xlang/fileio/fileio_test.go

```diff
@@ -0,0 +1,53 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: figure out how to connect this test to run with other integration tests.
+package fileio
```

Comment:
After running `./gradlew :sdks:java:io:file-schema-transform:expansion-service:runExpansionService`, I was able to run this test and saw on the java side "INFO: Expanding 'External' with URN 'beam:schematransform:org.apache.beam:file_write:v1'"